### PR TITLE
add tcp/tcp6 statistics

### PR DESCRIPTION
**Pull request by f0**
first working prototype for #889 , atm only workt for v1 api please review

### CODE REVIEW

File: container/libcontainer/helpers.go [hunk 1]
```diff
@@ -173,6 +187,78 @@ func scanInterfaceStats(netStatsFile string) ([]info.InterfaceStats, error) {
...
+	if err != nil {
+		return tcpStats, fmt.Errorf("couldn't read tcp stats: %v", err)
+	}
+
+	return tcpStats, nil
+}
+
+func scanTcpStats(tcpStatsFile string) (info.TcpStat, error) {
+
+	//FIXME besser lösen
```
Comment (vishh) [inline container/libcontainer/helpers.go]: Did you intend to add this comment?

File: info/v1/container.go [hunk 2]
```diff
@@ -347,6 +347,25 @@ type NetworkStats struct {
 	Interfaces     []InterfaceStats `json:"interfaces,omitempty"`
 }
 
+type TcpStat struct {
+	Established uint64
```
Comment (vishh) [inline info/v1/container.go]: Can you add a pointer that describes these fields?

**Comment by f0** [comment]
@vishh  ok updated the merge request. Any Idea how to get this working for v2?

File: info/v2/container.go [hunk 3]
```diff
@@ -109,6 +109,8 @@ type ContainerStats struct {
 	// Task load statistics
 	HasLoad bool         `json:"has_load"`
 	Load    v1.LoadStats `json:"load_stats,omitempty"`
+	//TCP statistics
+	TcpStat TcpStats `json:"tcpstat,omitempty"`
```
Comment (vishh) [inline info/v2/container.go]: Can we move this to NetworkStats defined below?

**Comment by vishh** [comment]
@rjnagal: What is pending to migrate over to v2 completely?

**Comment by rjnagal** [comment]
Moving UI to use v2 instead of v1.

**Comment by vishh** [comment]
Until UI transition happens, if we can make v2 REST API independent of v1
objects, that by itself will be useful. WDYT?

File: info/v2/container.go (hunk 3 above)
Comment (mcastilho) [inline info/v2/container.go]: I like this feature. Agreed, NetworkStats would be more suited for this stats.

**Comment by f0** [comment]
@vishh  moved to NetworkStats (for v1 and v2)

**Comment by jimmidyson** [comment]
@vishh :+1: to making v2 independent of v1 objects but that is out of scope for this PR.

**Comment by vishh** [comment]
@jimmidyson: My previous comment was for @rjnagal. This PR LGTM

**Comment by vishh** [comment]
ok to test

**Comment by jimmidyson** [comment]
LGTM. Merging... Thanks @f0!

**Pull request merged**